todo_app/app.py

```diff
@@ -4,46 +4,50 @@
 import todo_app.view_model as view_model
 from todo_app.Task import Task
 import todo_app.trello as trello
```

Comment:
We're no longer using Trello, so we can remove this import but also more broadly all of the references to Trello - we can always recover that with git if we need it